{
  "debianbug": 850716,
  "description": "XML External Entity (XXE) vulnerability in PySAML2 4.4.0 and earlier allows remote attackers to read arbitrary files via a crafted SAML XML request or response.",
  "releases": {
    "bullseye": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "bullseye": "4.5.0-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "buster": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "buster": "4.5.0-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "fixed_version": "2.0.0-1+deb8u1",
      "repositories": {
        "jessie": "2.0.0-1+deb8u1",
        "jessie-security": "2.0.0-1+deb8u2"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "sid": "4.5.0-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "stretch": "3.0.0-5"
      },
      "status": "resolved",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}